java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -2576,4 +2576,29 @@ public void pfmerge(BaseClient client) {
                         ExecutionException.class, () -> client.pfmerge(key1, new String[] {"foo"}).get());
         assertTrue(executionException.getCause() instanceof RequestException);
     }
+
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void objectFreq_returns_null(BaseClient client) {
+        String nonExistingKey = UUID.randomUUID().toString();
+        assertNull(client.objectFreq(nonExistingKey).get());
+    }
+
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void objectFreq(BaseClient client) {
+        String key = UUID.randomUUID().toString();
+        if (client instanceof RedisClient) {
+            assertEquals(
+                    OK, ((RedisClient) client).configSet(Map.of("maxmemory-policy", "allkeys-lfu")).get());
```

Comment:
should this policy be returned after the test so that it doesn't affect other tests?